autoinstrumentation/apache/README.md

```diff
@@ -0,0 +1,13 @@
+# How to build Apache auto-instrumentation docker image
```

Comment:
can we come up with a more explicit name (for the directory/image/readme name)? Apache is just too generic